app/views/curation_concerns/base/_attribute_rows.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= presenter.attribute_to_html(:description) %>
+<%= presenter.attribute_to_html(:creator, catalog_search_link: true ) %>
+<%= presenter.attribute_to_html(:contributor, label: 'Contributors', catalog_search_link: true) %>
+<%= presenter.attribute_to_html(:subject, catalog_search_link: true) %>
+<%= presenter.attribute_to_html(:publisher) %>
+<%= presenter.attribute_to_html(:language) %>
+<%= presenter.attribute_to_html(:identifier) %>
+<%= presenter.attribute_to_html(:tag) %>
+<!-- TODO: Rights is not being picked up. Is it indexed in Solr? -->
+<%= presenter.attribute_to_html(:rights) %>
```

Comment:
So the problem with the Rights is that the field is not being indexed into Solr, even thought Curation Concerns does index it. I am tempted to add a separate ticket for that.